gradle.properties

```
@@ -17,3 +17,11 @@ stdlibTimeVersion=2.0.0-beta.1-20210524-215500-0e4ffcc
 stdlibTransactionVersion=1.0.13-20210525-011300-87d3fb3
 stdlibUuidVersion=0.10.0-beta.1-20210524-221600-653a7c3
 stdlibCryptoVersion=1.1.0-beta.1-20210524-220100-ab68c07
+stdlibTaskVersion=2.0.0-beta.1-20210524-220600-b995270
+stdlibJWTVersion=1.1.0-beta.1-20210525-000500-26d9b99
+stdlibMimeVersion=1.1.0-beta.1-20210524-221400-441d6be
+stdlibHttpVersion=1.1.0-beta.1-20210525-154600-42de5aa
+stdlibOauth2Version=1.1.0-beta.1-20210525-000800-cf346d0
```

This might affect the automated dependency bumps.
```suggestion
stdlibOAuth2Version=1.1.0-beta.1-20210525-000800-cf346d0
```